src/main/java/com/mmaarten/mmcommands/MmCommandHandlerBuilder.java

```diff
@@ -0,0 +1,401 @@
+package com.mmaarten.mmcommands;
```

Comment:
Builder pattern requires this class to be a public static class inside of the MmCommandHandler.java so it can adhere to the OOP principles